aptos-move/framework/aptos-framework/doc/voting.md

```diff
@@ -107,8 +107,9 @@ Extra metadata (e.g. description, code url) can be part of the ProposalType stru
  Currently, we have three attributes that are used by the voting flow.
  1. RESOLVABLE_TIME_METADATA_KEY: this is uesed to record the resolvable time to ensure that resolution has to be done non-atomically.
  2. IS_MULTI_STEP_PROPOSAL_KEY: this is used to track if a proposal is single-step or multi-step.
- 3. IS_MULTI_STEP_PROPOSAL_IN_EXECUTION_KEY: this attribute only exists for and applies to multi-step proposals. The value is used to
- indicate if a multi-step proposal is in execution. If yes, we will disable further voting for this multi-step proposal.
+ 3. IS_MULTI_STEP_PROPOSAL_IN_EXECUTION_KEY: this attribute only applies to multi-step proposals. A single-step proposal will not have
```

Comment:
Create a separate PR for this? Seems irrelevant change

Reply:
https://github.com/aptos-labs/aptos-core/pull/5815